### DISCUSSION TITLE
[Atlantis] Component Dialog. Visualisation of focus on the close button

### DISCUSSION THREAD

**Discussion by nk2048** [primefaces/community#86] [PrimeVue Templates] [answered]
Hi,

It would be great when the focused close button on dialog gets highlighted.
On the image below the close button is focused but nobody can see it. (Atlantis version 5.1.0)
![atlantis-dialog-closebutton](https://github.com/primefaces/community/assets/57187499/1b933ea3-59b4-45e3-8f57-a0874adb7d1a)

The expectation is when the close button is focused, it can be seen.
![atlantis-dialog-closebutton2](https://github.com/primefaces/community/assets/57187499/bc160e8d-e3fb-4b58-903e-b6072e77f8dc)

Thanks!

**Comment by tugcekucukoglu** [comment, marked as answer]
The below code can be added here `public/theme/theme-light/_extension.scss`

```
.p-dialog .p-dialog-header .p-dialog-header-icon:focus {
    outline: 0 none;
    outline-offset: 0;
    box-shadow: 0 0 0 0.2rem #BFDBFE;
}
```

**Reply by nk2048** [reply]
@tugcekucukoglu Thank you for the quick reply. It works!

**Comment by nk2048** [comment]
Will you fix this issue in the next version of Atlantis?

**Comment by tugcekucukoglu** [comment]
Actually, this is not a bug, it is a design decision. For more information: https://github.com/primefaces/primevue/issues/4500
